app/assets/javascripts/components/course_queue.js.jsx

```diff
@@ -246,6 +246,9 @@ var CourseQueue = React.createClass({
           emptyQueue={this.handler.emptyQueue.bind(this.handler)}
           setInstructorStatus={this.handler.setInstructorStatus.bind(this.handler)}
           takeQueueOffline={this.handler.takeQueueOffline.bind(this.handler)}
+          pin={this.handler.pin.bind(this.handler)}
+          getUnpinnedRequests={this.getUnpinnedRequests()}
```

Comment:
nit: this is really just `unpinnedRequests` since you're invoking the function here and not passing a callable down to `InstructorPanel`